meilenstein_02/analyse_Speichermedium/README.md

```diff
@@ -0,0 +1,103 @@
+    - Es ist möglich Dateien zu erstellen für verschiedene Datensätze der Funktionen von verschiedenen Versuchen
+    - Es wird dem Dataclient eine Klasse mit Funktionen zur Verfügung gestellt, welche ihm ermöglichen nach Informationen, Dateien, als auch ganzen Datenordnern zu suchen
+    - Das System ermöglicht es Datensätze auch wieder zu löschen
+
+## Was ist der erwartete Input?
+    Es werden Datensätze von den Funktionen *Zoom Maps*, *Codecharts*, *Webcam basiertes Eyetracking* und DataClient erwartet. 
+
+## Was ist der erwartete Output?
+    Es wird erwartet, dass der DataClient nach Dateien, Ordnern, als auch einzelnen Informatione fragt und diese ihm dann ausgegeben werden müssen.
+
+## Was sind Nomen Phrasen?
+<!-- Alle relevanten Sachen aufschreiben, später kann aussortiert werden! -->
+    - Speicher/Schreiben
+    - Lesen/Suchen
+    - Löschen/Aendern
+    - Sortieren
+
+## Liste der Klassen:
+<!-- Erstmal alle aufschreiben, dann auswählen! (Kriterien siehe Vorgehensweise) -->
+<!-- Warum sind die Klassen existent? Wenn das zu beantworten ist - u good! -->
+<!-- ausgewählte Klassen mit Link, andere einklammern und CRC-Karte löschen -->
+- [Suchen](crc-{Suchen}.md)
+- [Speichern](crc-{Speichern}.md)
+- [Aendern](crc-{Aendern}.md)
+- (Sortieren)
+
+---
+
+# Analyse - Speichermedium
+<!-- Hier Notizen zum Denkprozess! -->
+1. Welche Aufgaben hat das Speichermedium/soll es zur Verfügung stellen?
+     - ***Es muss zuerst die Art des Speichermediums entschieden werden***.. (Datenbank oder Datei) ..
+     ***Da dies noch in Besprechung ist, gehe ich aktuell von eine Datei aus.***
+     - Neben dem **Als was** etwas abgespeichert werden soll gibt es noch andere Anforderungen
+     - Es existiert eine Abstrakte Oberklasse **Speichefunktionen* die in Verbindungen mit den Funktionen an sich steht
+     - Die Funktionen die zum aktuellen Zeitpunkt zur Verfügung gestellt werden sollen, sind **Speichern/Schreiben**, **Suchen/Lesen** und **Ändern**
+     - Damit ist gemeint, dass die Funktion **Suchen** dem *DataClient* zur Verfügung stehen soll und die Funktionen **Speichern** und **Löschen/Aendern**, den Funktionen *Zoom Maps*, *Codecharts*, *Webcam basiertes Eyetracking* und *ConfigFile*
+     - Da die 3 großen Funktionen aus mehreren Unterfunktione besteht, werden diese zu jeweils 3 Klassen
+2. Was macht die Klasse ***Suchen*** ?
+     - Diese muss nur dem DataClient zur Verfügung gestellt werden
+     - DataClient kann speziell nach Informationen suchen
+     - Besitzt für alle 3 Funktionen spezielle Suchen, für die jeweiligen Daten
+     - Besitzt die Möglichkeit nach Dateinnamen als auch nach allen Daten von einer Funktion zu suchen
+     - Kann einzelne Daten (falls notwendig) als auch einen kompletten Datensatz ausgeben
+3. Was macht die Klasse ***Speichern***
+     - Diese Klasse steht den drei Funktionen, sowie der ConfigFile zur Verfügung
```

Comment:
Die Kollaboration mit dem ConfigFile ist in dem CRC zu Speichern nicht festgehalten, wenn mich nicht alles täuscht

Reply:
Würde ich auch nicht, siehe #25
würde ich als Notiz unten in der readme anfügen, sonst gar nicht ^^